src/main/java/lotto/Application.java

```diff
@@ -1,7 +1,128 @@
 package lotto;
 
+import camp.nextstep.edu.missionutils.Console;
+import camp.nextstep.edu.missionutils.Randoms;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
 public class Application {
+
+
     public static void main(String[] args) {
-        // TODO: 프로그램 구현
+        int inputMoney = 0;
+        int totalPrize = 0;
+        int firstCount = 0;
+        int secondCount = 0;
+        int thirdCount = 0;
+        int fourthCount = 0;
+        int fifthCount = 0;
+        List<Integer> inputWinningNumber = new ArrayList<>(); // ArrayList로 초기화
+        int inputBonusNumber=0;      //try를 사용해서 변수를 위에다가 다 선언
+
+        {
+            try {
+                System.out.println("구입금액을 입력해 주세요.");
+                inputMoney = Integer.parseInt(Console.readLine());
+                if (inputMoney < 1000) {
+                    throw new IllegalArgumentException("[ERROR] 구입금액은 1000원이상이여야합니다");
+                } else if (inputMoney % 1000 != 0) {
+                    throw new IllegalArgumentException("[ERROR] 구입금액은 1000원 단위여야합니다");
+                }
+
+            } catch (NumberFormatException e) {
+                System.out.println("[ERROR] 잘못된 입력입니다 숫자를 입력해주세요");
+                System.exit(1);
+            } catch (IllegalArgumentException e) {
+                System.out.println(e.getMessage());
+                System.exit(1);
+            }
+
+            // 로또 생성
+            int numberOfLotto = inputMoney / 1000;
+            List<List<Integer>> purchasedLotto = new ArrayList<>(); //로또를 여러개 사야하니깐
+            System.out.println(numberOfLotto + "개를 구매하였습니다.");
+            for (int i = 0; i < numberOfLotto; i++) {
+                List<Integer> lottoNumbers = Randoms.pickUniqueNumbersInRange(1, 45, 6);
+                purchasedLotto.add(lottoNumbers);
+                List<Integer> sortedNumbers = lottoNumbers.stream()
+                        .sorted()
+                        .toList();
+                System.out.println(sortedNumbers);
+            }
+
+            //값 입력
+            try {
+                System.out.println("당첨 번호를 입력해주세요");
+                String input = Console.readLine();
+                inputWinningNumber = Arrays.stream(input.split(","))
+                        .map(String::trim)
+                        .map(Integer::parseInt)
+                        .toList();
+                if(inputWinningNumber.size() != 6) {
+                    throw  new IllegalArgumentException("[ERROR] 로또 번호는 6개를 입력해야합니다");
+                    }
+                for (int winningNumber : inputWinningNumber) {
+                    if (!(1 <= winningNumber && winningNumber <= 45)) {
+                        throw new IllegalArgumentException("[ERROR] 당첨번호는 1~45사이의 숫자입니다");
+                    }
```

Comment:
```suggestion
                for (int winningNumber : winningNumbers) {
                    if (!(1 <= winningNumber && winningNumber <= 45)) {
                        throw new IllegalArgumentException("[ERROR] 당첨번호는 1~45사이의 숫자입니다");
                    }
```
다수의 당첨번호를 담고 있는 리스트 이름을 winningNumbers 로 지으면, 이해가 더 잘 되지 않을까? 🙂

Reply:
이 배열은 입력받은 당첨번호를 담고있다고 생각해서 이렇게 적는게 더 이해가 잘될것 같다고 생각해서 바꾸지 
않았습니다!

Reply:
> 이 리스트는 입력받은 당첨번호를 담고있다고 생각해서 이렇게 적는게 더 이해가 잘될것 같다고 생각해서 바꾸지 않았습니다!